config/locales-glitch/en.yml

```diff
@@ -88,6 +88,10 @@ en:
   generic:
     use_this: Use this
   invites:
+    errors:
+      account_age: Your account isn't old enough to create invites
```

Comment:
This error is actually never shown, since you can't even navigate to invites. Oops.